### ISSUE TITLE
Multiple RTSP streams are not managed

### ISSUE THREAD

**Issue by fuatakgun** [bropat/eufy-security-client#63]
**Describe the bug**  
I can send commands to start RTSP on multiple cameras and they are working fine but when I need to stop, only last one is succeeded and initial ones are throwing errors.

**To Reproduce**  
```set_rtsp_stream serial_number_1 True -> rtsp_livestream_started event generated
set_rtsp_stream serial_number_2 True -> rtsp_livestream_started event generated
set_rtsp_stream serial_number_1 True -> RTSPLivestreamNotRunningError  RTSP livestream for device serial_number_1 could not be stopped, because it is not running
set_rtsp_stream serial_number_2 True -> rtsp_livestream_stopped event generated
```

**Expected behavior**  
As RTSP is managed by devices, there is no reason to limit number of parallel streams. They should all be playing independently.

**Versions:**  
Eufy Security WS 0.5.2

**Comment by bropat** [comment]
As already mentioned in another issue, I cannot reliably check whether an RTSP stream is running or not (Eufy simply does not communicate this). I can only provide the status if you have started the RTSP stream via the eufy-security-client API.

What I can do, is simply remove the exceptions for RTSP start/stop livestream (on start: livestream already running; on stop: livestream not running) on eufy-security-ws and that's it.

Unless you have a better idea.

**Comment by bropat** [comment]
> What I can do, is simply remove the exceptions for RTSP start/stop livestream (on start: livestream already running; on stop: livestream not running) on eufy-security-ws and that's it.

In version 0.5.3 I did it as described above

**Comment by fuatakgun** [comment]
it makes sense what you have done but it is still not raising events for stream stopped.

i can see that starting and stopping rtsp are actually working fine as it was working fine in previous version and no exception is thrown, we still need an event to be raised after stopping rtsp.

**Comment by bropat** [comment]
@fuatakgun 

> it makes sense what you have done but it is still not raising events for stream stopped.

This issue concerns the error above and not the event problem you describe. Please do not mix things.

> i can see that starting and stopping rtsp are actually working fine as it was working fine in previous version and no exception is thrown, we still need an event to be raised after stopping rtsp.

I don't agree looking at the following error you posted above:

```
set_rtsp_stream serial_number_1 True -> RTSPLivestreamNotRunningError  RTSP livestream for device serial_number_1 could not be stopped, because it is not running
```

**Comment by bropat** [comment]
@fuatakgun 

One more thing: get used to providing the debug logs

**Comment by fuatakgun** [comment]
I see that you are focusing on specific sentences rather than underlying ask. This issue is about `managing multiple RTSP streams` not about specifically raising or removing errors.

You can try to extract the problem, rather than I mix the things;
```
set_rtsp_stream serial_number_1 True -> rtsp_livestream_started event generated
set_rtsp_stream serial_number_2 True -> rtsp_livestream_started event generated
set_rtsp_stream serial_number_2 False -> rtsp_livestream_stopped event generated
set_rtsp_stream serial_number_1 False -> No event generated
```

**Comment by fuatakgun** [comment]
Logs from client

```
device.start_rtsp_livestream T8113N63205018FC
eufy-security> {
  type: 'event',
  event: {
    source: 'device',
    event: 'rtsp livestream started',
    serialNumber: 'T8113N63205018FC'
  }
}
{
  type: 'result',
  success: true,
  messageId: 'start_rtsp_livestream',
  result: {}
}
device.start_rtsp_livestream T8113N63205014E2
eufy-security> {
  type: 'event',
  event: {
    source: 'device',
    event: 'rtsp livestream started',
    serialNumber: 'T8113N63205014E2'
  }
}
{
  type: 'result',
  success: true,
  messageId: 'start_rtsp_livestream',
  result: {}
}
device.stop_rtsp_livestream T8113N63205018FC
eufy-security> {
  type: 'event',
  event: {
    source: 'device',
    event: 'rtsp livestream stopped',
    serialNumber: 'T8113N63205014E2'
  }
}
{
  type: 'result',
  success: true,
  messageId: 'stop_rtsp_livestream',
  result: {}
}
device.stop_rtsp_livestream T8113N63205014E2
eufy-security> {
  type: 'result',
  success: true,
  messageId: 'stop_rtsp_livestream',
  result: {}
}
```

**Comment by bropat** [comment]
> I see that you are focusing on specific sentences rather than underlying ask. This issue is about `managing multiple RTSP streams` not about specifically raising or removing errors.

Look at your description:

Describe the bug
I can send commands to start RTSP on multiple cameras and they are working fine but when I need to stop, only last one is succeeded and initial ones are throwing errors.

**Comment by bropat** [comment]
> Logs from client

I will check this out.

**Comment by fuatakgun** [comment]
As I said, feel free to consume the way you prefer, as the initial error was about 0.5.2 and you released 0.5.3 which removed exception raising, but did not fix the expected result.

Attached the server logs.

[logs.zip](https://github.com/bropat/eufy-security-client/files/7553616/logs.zip)
